aif360/algorithms/inprocessing/gerryfair/__init__.py

```diff
@@ -1,2 +1,3 @@
+from aif360.algorithms.inprocessing.gerryfair_classifier import GerryFairClassifier
```

Comment:
actually, you needed to add this line to `aif360/algorithms/inprocessing/__init__.py` -- one directory above this